examples/data-objects/inventory-app/src/inventoryList.ts

```diff
@@ -7,27 +7,51 @@ import { DataObject, DataObjectFactory } from "@fluidframework/aqueduct";
 import {
 	AllowedUpdateType,
 	ForestType,
-	TypedTreeChannel,
-	TypedTreeFactory,
+	ISharedTree,
+	ISharedTreeView,
+	SharedTreeFactory,
 	typeboxValidator,
 } from "@fluid-experimental/tree2";
 import { IFluidHandle } from "@fluidframework/core-interfaces";
-import { InventoryField, schema } from "./schema";
+import { Inventory, schema } from "./schema";
 
 const treeKey = "tree";
 
-const factory = new TypedTreeFactory({
+const factory = new SharedTreeFactory({
 	jsonValidator: typeboxValidator,
 	forest: ForestType.Reference,
-	subtype: "InventoryList",
 });
```

Comment:
TypedTreeFactory/TypedTreeChannel only exposes the ET2 root field.  Reverting to SharedTreeFactory for the time being to access ISharedTree -> ISharedTreeView -> proxy root.